.github/scripts/create-values-diff.sh

```diff
@@ -45,7 +50,7 @@ trap 'rm -rf "$TMP_DIR"' EXIT
 cd "$GITHUB_WORKSPACE"
 
 function generateComment() {
-  local chart="charts/${1?}"
+  local chart="${1?}"
```

Comment:
I cannot see why the path changed from charts/$ {1} to just {$1}

Reply:
Because he changed the way it's called.

You can see this here: https://github.com/teutonet/teutonet-helm-charts/pull/854/files#diff-1c060ea3361efbeee5f55dc7e6086ed9cdf182099274e382d630cac8a092dc60R28

Reply:
My guess is, he did it to make it more universal. Or is there a different reason @cwrau?